src/services/review/RepoCheckerService.ts

```diff
@@ -346,24 +418,56 @@ export default class RepoCheckerService {
         .commit("site checker logs")
         .push(),
       (error) => new SiteCheckerError(`${error}`)
-    )
+    ).map(() => errors)
   }
 
-  checkRepo(repo: string): ResultAsync<undefined, SiteCheckerError> {
+  checkRepo(repo: string): ResultAsync<RepoErrorDto, RepoErrorDto> {
```

Comment:
hmm, this looks abit weird - should we return the same type of error if it succeeds?

Reply:
wah ok i clearly need to rename this then. The error here was trying to refer to the errors found in the repo which we are going to report to the user, not an error in the cms. will rename it to BrokenLinkErrorDto instead